Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/EndRide/Internal.hs

```diff
@@ -295,6 +332,17 @@ getCurrentDate time =
   let currentDate = localDay $ utcToLocalTime timeZoneIST time
```

Comment:
All these type of functions are not correct, we'll have to take for US. cc: @hkmangla